src/minerfund.cpp

```diff
@@ -24,15 +24,30 @@ static const CTxOut BurnOutput(const Amount amount) {
     return {amount, CScript() << OP_RETURN << burnPayload};
 }
 
+std::vector<std::string> citizenAddresses = {
+    "pqnqv9lt7e5vjyp0w88zf2af0l92l8rxdgnlxww9j9",
+    "qqrad726sy24klmsd4dus5gxl0rhgz0rdcjnagpmve",
+    "qzafzyamh8rgszacw73gg2vly8gnxe099qx6fqrdfd",
+};
+
+static CTxOut BuildWinnerOutput(uint64_t slot, Amount shareAmount, uint256 seed) {
```

Comment:
Should we call it `epochBlockHash` instead of `seed`?

I have to admit I'm quite delighted that we already found a really good use for the epoch hash. Imagine how much more difficult the implementation of this would be like without it just in the header.

And certainly more uses to come.

Reply:
Also, if possible, we shouldn't use jargon like "winner". It should be more like "selected" or "chosen" or "representative". This is not a lottery nor raffle, so we shouldn't use terms that could confuse regulators.

Reply:
Sure